src/main/java/org/broadinstitute/hellbender/engine/VariantWalker.java

```diff
@@ -58,7 +58,7 @@ protected void initializeDrivingVariants() {
         //Add the driving datasource to the feature manager too so that it can be queried. Setting lookahead to 0 to avoid caching.
         //Note: we are disabling lookahead here because of windowed queries that need to "look behind" as well.
         features.addToFeatureSources(0, drivingVariantsFeatureInput, VariantContext.class, cloudPrefetchBuffer, cloudIndexPrefetchBuffer,
-                                     referenceArguments.getReferencePath());
+                                     referenceArguments.getReferencePath(), errorOnOutOfDateIndex);
```

Comment:
You are still missing a bunch of places in the engine where a `FeatureDataSource` is constructed. For example, a few lines above this line, a `FeatureDataSource` is constructed for the `drivingVariants`, but the `errorOnOutOfDateIndex` setting is not propagated to the constructor.

Recommend doing a find usages on the `FeatureDataSource` constructors to catch the rest of these (should do the same for `ReadsDataSource` as well.